var/spack/repos/builtin/packages/mofem-cephas/package.py

```diff
@@ -24,15 +24,34 @@
 
 
 class MofemCephas(CMakePackage):
-    """mofem-cephas core library"""
+    """MoFEM is finite element core library"""
 
     homepage = "http://mofem.eng.gla.ac.uk"
-    git      = "https://bitbucket.org/likask/mofem-cephas.git"
+    git = "https://bitbucket.org/likask/mofem-cephas.git"
 
     maintainers = ['likask']
 
-    version('develop', branch='develop', submodules=True)
-    version('0.8.7', tag='v0.8.7', submodules=True)
+    version('0.8.15', git='https://bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.8.15')
+    version('0.8.14', git='https://bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.8.14')
+    version('0.8.13', git='https://bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.8.13')
+    version('0.8.12', git='https://bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.8.12')
+    version('0.8.11', git='https://bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.8.11')
+    version('0.8.10', git='https://bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.8.10')
+    version('0.8.9', git='https://bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.8.9')
+    version('0.8.8', git='https://bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.8.8')
+    version('0.8.7', git='https://bitbucket.org/likask/mofem-cephas.git',
+        tag='v0.8.7')
+    version('develop',
+        git='https://bitbucket.org/likask/mofem-cephas.git',
+        branch='develop')
```

Comment:
Since you have a package-level `git` variable, you no longer need to specify `git` for each version. Same for most of your other packages.

Reply:
Fixed.